include/h2o.h

```diff
@@ -471,6 +471,14 @@ struct st_h2o_globalconf_t {
          * a boolean indicating if UDP GSO should be used when possible
          */
         uint8_t use_gso : 1;
+        /**
+         * if handshake does not complete within RTT multiplied by this value, we'll close a connection
+         */
+        uint32_t handshake_timeout_rtt_multiplier;
+        /**
+         * if the number of Initial/Handshake packets sent exceeds this limit, treat it as an error and close a connection
+         */
+        uint64_t max_initial_handshake_packets;
```

Comment:
How about making these parameters an attribute of `listen` -> `quic` node, alongside other config knobs such as `retry`, `max-udp-payload-size`?

Reply:
Sounds like a good idea. Addressed in 479c20592.